express-delete/server.ts

```diff
@@ -0,0 +1,56 @@
+import express from 'express';
+
+interface Student {
```

Comment:
I prefer to use `type`. I know the TS docs suggest using `interface` unless you need `type`, but I think it's wrong. There are many reasons `type` is better, and I frequently end up needing the functionality of `type`, anyway, so for consistency and ease of programming, I always use `type`. (Except in those extremely rare cases where I need `interface`. I would say that in the past 4 years I've used `interface` once.)